vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-#cd oms-docker/docker
-#make bootstrap
 
 cd /vagrant/
 #possible to use flag -d
-./oms.sh up -d
\ No newline at end of file
+make bootstrap
+sleep 90 #to give time for the bootstrap
```

Comment:
I suggest adding proper healthcheck here, something like

```
while (true) {
  if (fetch(oms_core_elixir_address)) {
    break;
  }

  sleep(2);
}`

(no idea how it would be in bash, so it is pseudo JS).
Not critical though, but would be nice, so we'd know if everything has really launched.

Reply:
I wanted to avoid potentially infinite loops (if core is broken, e.g.)

Also, the address of frontend should be there, not of core? I guess curl is the real world implementation of what you suggested

Reply:
1) we can add some threshold to exit it, e.g. 5 minutes
2) I'd say the backend. For example, the frontend is loaded but the backend is still loading (was the case for me at least, as somehow building oms-core-elixir takes a lot of time). Then the system will be still unusable till oms-core-elixir will start

Reply:
I thought anyway that we should add some Healthcheck in the services (it was in my mind since long), to both know about their health and to accommodate situations like this one (and swarm, or consul, or portainer, or whatever like curl can know how to assess container's health).

Just a simple endpoint like `/ping` or `/health`

Reply:
For now all it does is to check if I got the `<html>` tag, which is the error that Phoenix gives because it is not a known route

Reply:
Also, I do that by changing the definition in `docker-compose.yml` of oms-core-elixir from `expose` to `ports`... i.e. I would need a PR on it now